src/features/pages/markets/etf/instruments-etf/data.tsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react'
+import { Localize } from 'components/localization'
+import { InstrumentContentType } from 'features/components/molecules/instruments-table/types'
+import * as icons from 'components/elements/symbols'
+
+export const tableContent: InstrumentContentType = {
+    mobile_basis: '1-2',
+    desktop_basis: '1-3',
+    title: '_t_ETFs_t_',
+    data: [
+        {
+            id: 0,
+            src: icons.STOCKSUS500,
+            text: <Localize translate_text="_t_AGG.US_t_" />,
+        },
+        {
+            id: 1,
+            src: icons.STOCKSUS500,
+            text: <Localize translate_text="_t_ARRK.US_t_" />,
```

Comment:
can you pass the `Localize` in the component and just pass the data here

Reply:
I refer to this page,   similar usage, can you please check and let me know?
src/pages/markets/static/content/_market-symbols.tsx

Reply:
we are making it as reusable component right.so we can use Localize component inside the reusable component and just pass the data.Instead of calling Localize component in all places